wordpress-mobile/WordPress-iOS
# Site Management UI

Adds the UI for our first pass at WordPress.com site management features as listed in #4570.

To Test:
- Check that Blog > Settings now has an 'Advanced' section with 'Start Over' and 'Delete Site' items for WordPress.com blogs that user has admin access to
- Check that section is not there for non-WordPress.com blogs or non-administered blogs
- Check that Start Over controller opens a Helpshift session if enabled, and goes to browser support otherwise (similar to behaviour of Me > Support)
- Check that Delete Site controller correctly displays blog's primary domain and requires exact match to enable deletion 
- Verify that a deleted site is in fact deleted

Please Review: @kurzee
